developer-community/tech-talks

```diff
@@ -0,0 +1,202 @@
+# The full story
+
+## Speakers
+##### Contact
+
+Ideally, we should start contacting speakers __2.5 months__ prior to the event.
+
+Our tech talks involve a total of __5 long talks__ and __2 lightning__. Someone from our team should always give either one Lightning or a Long talk. Hence, we should make an initial list of _6 people to contact_. If no reply, _follow-up after 3-5 days_ until you get an answer. 
+
+Email copy:
+
+> Hello Jessica,
+> My name is Sergio Arbeo, we met last year at EmberFest.
+> I am working at source{d} and we are organising a one-day conference focused on front-end technologies. Please, feel free to take a look at our past edition, around Infrastructure, on our website (videos will be up in the next few days).
+> We saw your talk Animate the Web with Ember.js and this is exactly what we are looking for: an advanced technical talk that, while focusing on a particular framework, is useful for the community as a whole.
+> Talks are 45min long (~30min for talk, ~15min for Q&A, we have 15min between talks, so no need to hurry) and the topic is up to you as long as it is advanced / new and does not exclude anyone based on just the framework they use. Your talk at EmberConf is a perfect example of what we are aiming at.
```

Comment:
Remove name. Remember that the talk is in English if contacting a Spanish speaking person.